web/packages/teleport/src/services/api/parseError.ts

```diff
@@ -29,6 +52,12 @@ export default function parseError(json) {
   return msg;
 }
 
+export function parseProxyVersion(json): ProxyVersion | null {
```

Comment:
This won't return `null` but `undefined`.
```suggestion
export function parseProxyVersion(json): ProxyVersion | undefined {
```